Sources/Views/AnimatableTableViewCell.swift

```diff
@@ -156,7 +159,9 @@ open var startPoint: GradientStartPoint = .top
   fileprivate func configureInspectableProperties() {
     configureAnimatableProperties()
     configureOpacity()
+    #if os(iOS)
```

Comment:
If we put the checking `#if os(iOS)` within `configureSeparatorMargins()` doesn't it help to eliminate the checking int he subclass? The subclass still calls it, and on tvOS, it doesn't do anything by default.